src/languages/marko.coffee

```diff
@@ -1,8 +1,15 @@
+# Get Atom defaults
+scope = ['text.marko']
+tabLength = atom?.config.get('editor.tabLength', scope: scope) ? 4
+softTabs = atom?.config.get('editor.softTabs', scope: scope) ? true
+defaultIndentSize = (if softTabs then tabLength else 4)
+defaultIndentChar = (if softTabs then " " else "\t")
+defaultIndentWithTabs = not softTabs
+
 module.exports = {
 
   name: "Marko"
   namespace: "marko"
-  fallback: ['html']
```

Comment:
Are you sure you do not want to fallback to `html` settings? The settings in the `makro` namespace will take precedence but in the case that `makro` is not set then `html` will be used.